packages/markdown/html.lua

```diff
@@ -0,0 +1,44 @@
+local base = require("packages.base")
+local utils = require("packages.markdown.utils")
+
+local package = pl.class(base)
+package._name = "markdown.html"
+
+local passthroughs = {
+  "sup",
+  "sub",
+  "em",
+  "strong",
+  { ["div"] = "hbox" },
```

Comment:
A div cannot be made an hbox (or it won't be properly line broken) -- same for a span actually. 

But more generally, Pandoc has lots of extensions regarding HTML in Markdown, e.g. `markdown_in_html_blocks` which allows mixing HTML and Markdown, if I read that correctly:
```
I have one more concern which comes to mind: I'd ideally like both the **pandocast** and **markdown** packages to have the same minimal set of features. This imply, for any test we write, looking at what the Pandoc AST would contain, to at least ensure reasonable parity, whichever route is takent.